main.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+// Great help
+// https://golangcode.com/how-to-read-a-csv-file-into-a-struct/
+
+import (
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+)
+
+const csvFilename = "pokemons.csv"
+
+type Pokemon struct {
```

Comment:
By convention, every exported stuff (function, variable, constant, etc) should have a description

```go
// Pokemon - Description here....
type Pokemon struct {
```